#! /bin/sh /usr/share/dpatch/dpatch-run
## 15-fix-lzo-flags.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: hardcode LZO headers location.

@DPATCH@

diff -rupd ACE_wrappers.orig/MPC/config/lzo1.mpb ACE_wrappers/MPC/config/lzo1.mpb
--- ACE_wrappers.orig/MPC/config/lzo1.mpb	2011-02-13 20:54:12.115281001 +0100
+++ ACE_wrappers/MPC/config/lzo1.mpb	2011-02-13 20:54:51.187281001 +0100
@@ -8,11 +8,11 @@ feature(lzo1) {
     LZO1_INCDIR
     $(LZO1_ROOT)/include
   }
-  includes += $(LZO1_INCDIR)
+  includes += /usr/include/lzo
   libpaths += $(LZO1_ROOT)/lib
   specific (prop:microsoft) {
     lit_libs += liblzo
   } else {
-    lit_libs += lzo
+    lit_libs += lzo2
   }
 }
